app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+from app.players.player import Player

```

Comment:
fix imports 

![Screenshot from 2023-10-21 17-41-12](https://github.com/mate-academy/py-elves-and-dwarves/assets/122517203/f135f136-b648-4cfb-8f7d-dd66782f2b32)